utils/copyImagesToQuay.txt

```diff
@@ -0,0 +1,20 @@
+# list images to copy to quay.io/eclipse/che-*, 1 per line
+# lines starting with # will be ignored 
+# see copyImagesToQuay.sh for skopeo copy script
+
+docker.io/centos/mongodb-36-centos7
+docker.io/centos/mysql-57-centos7
+docker.io/centos/postgresql-96-centos7
+docker.io/dirigiblelabs/dirigible-openshift:3.4.0
+docker.io/ksmster/che-editor-jupyter:5.7.0
+docker.io/traefik:v2.2.8
+docker.io/wsskeleton/eclipse-broadway
+
+# chinodesuuu/coder has moved 
+# docker.io/chinodesuuu/coder:2.1523-vsc1.38.1-che
```

Comment:
It's a plugin provided by community it could stay on docker.io

Reply:
except that it'll cause user experience failures so we should copy it over to quay.

Reply:
meanwhile https://quay.io/repository/eclipse/che--codercom--code-server

Reply:
It probably won't cause any pain on these one.but it will cause pain to maintain these useless forks

Reply:
100% agree, but making upstream projects publish to quay instead of docker is something we can tackle in the future.

Reply:
I doubt that release is blocked for plugin's images

Reply:
this image should be removed from the script